cdap-app-fabric/src/main/java/io/cdap/cdap/gateway/handlers/AppLifecycleHttpHandler.java

```diff
@@ -419,7 +415,7 @@ public void deleteAppVersion(HttpRequest request, HttpResponder responder,
                                @PathParam("app-id") final String appId,
                                @PathParam("version-id") final String versionId) throws Exception {
```

Comment:
What was the final decision, do we allow this cherry-delete? @chtyim ?

Reply:
yes, we do this https://github.com/cdapio/cdap/pull/14502, plan to merge this after this PR